pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -345,6 +348,7 @@ func (c *Controller) watchNetworkPolicies() {
 					return
 				}
 				klog.V(2).Infof("Added NetworkPolicy (%#v)", event.Object)
+				klog.Infof("NetworkPolicy %s(%s/%s) applied to Pods on this Node", policy.UID, policy.Namespace, policy.Name)
```

Comment:
Do you think the UID is useful in these log messages?

Reply:
As I have added policy namespaced name to all logs across controller and agent, the UID will be no longer needed. Removed.